online-store/src/components/routing/routing.ts

```diff
@@ -0,0 +1,300 @@
+import { searchProductCard } from './../search/search';
+import Router from 'vanilla-router';
+import './404.scss';
+import { renderDetails } from '../details/productCard';
+import { itemInBasket } from '../..';
+import { productsData } from '../cards/cards';
+import { renderBasket } from '../basket/basket';
+import { productsArrayI } from '../types';
+import { changeFilter } from '../filter/filter';
+import { localStorageUrl } from '../basket/localStorage';
+
+export function recoveryValue(value: string) {
+  const stockLowerSlider = document.querySelector(
```

Comment:
the name of the module suggests that there will be routing here, it is not entirely clear how this function is related to routing